src/config.rs

```diff
@@ -185,7 +207,7 @@ pub struct SharedConfigValues {
     password: Option<String>,
 
     /// Enables keyring password access
-    #[structopt(long)]
+    #[cfg_attr(feature = "dbus_keyring", structopt(long))]
     #[serde(alias = "use-keyring", default, deserialize_with = "de_from_str")]
```

Comment:
Does it make sense to add the cfg_attr to the serde field too? In theory it wouldn’t change anything as the check only happens if the feature I set anyway. But it would be more consistent behavior as the field can’t be modified in any way if the feature is not available

Reply:
Yeah, that makes sense.